plugins/woocommerce/src/Internal/Admin/Reviews.php

```diff
@@ -162,4 +174,132 @@ public function render_reviews_list_table() {
+	 *     @type int $moderated      The number of comments awaiting moderation (a.k.a. pending).
+	 *     @type int $spam           The number of spam comments.
+	 *     @type int $trash          The number of trashed comments.
+	 *     @type int $post-trashed   The number of comments for posts that are in the trash.
+	 *     @type int $total_comments The total number of non-trashed comments, including spam.
+	 *     @type int $all            The total number of pending or approved comments.
+	 * }
+	 */
+	public function count_comments( $count = [], $post_id = 0 ) {
+
+		$post_id = (int) $post_id;
+
+		/**
+		 * Filters the comments count for a given post or the whole site.
+		 *
+		 * @param array|stdClass $count   An empty array or an object containing comment counts.
+		 * @param int            $post_id The post ID. Can be 0 to represent the whole site.
+		 */
+		$filtered = apply_filters( 'woocommerce_product_reviews_count_comments', array(), $post_id );
+		if ( ! empty( $filtered ) ) {
+			return $filtered;
+		}
+
+		$count = wp_cache_get( "comments-{$post_id}", 'counts' );
+		if ( false !== $count ) {
+			return $count;
+		}
+
+		$stats              = $this->get_comment_count( $post_id );
+		$stats['moderated'] = $stats['awaiting_moderation'];
+		unset( $stats['awaiting_moderation'] );
+
+		$stats_object = (object) $stats;
+		wp_cache_set( "comments-{$post_id}", $stats_object, 'counts' );
+
+		return $stats_object;
+	}
+
+	/**
+	 * Retrieves the total comment counts for the whole site or a single post.
+	 *
+	 * This method overrides the default get_comment_count from WordPress to not consider product reviews as comments.
+	 *
+	 * @param int $post_id Optional. Restrict the comment counts to the given post. Default 0, which indicates that
+	 *                     comment counts for the whole site will be retrieved.
+	 * @return int[] {
+	 *     The number of comments keyed by their status.
+	 *
+	 *     @type int $approved            The number of approved comments.
+	 *     @type int $awaiting_moderation The number of comments awaiting moderation (a.k.a. pending).
+	 *     @type int $spam                The number of spam comments.
+	 *     @type int $trash               The number of trashed comments.
+	 *     @type int $post-trashed        The number of comments for posts that are in the trash.
+	 *     @type int $total_comments      The total number of non-trashed comments, including spam.
+	 *     @type int $all                 The total number of pending or approved comments.
+	 * }
+	 */
+	protected function get_comment_count( $post_id = 0 ) {
+		global $wpdb;
+
+		$post_id = (int) $post_id;
+
+		$where = '';
+		if ( $post_id > 0 ) {
+			$where = $wpdb->prepare( $where . ' AND comment_post_ID = %d', $post_id ); // phpcs:ignore WordPress.DB.PreparedSQL.NotPrepared
+		}
+
+		$sql = "
+			SELECT comment_approved, COUNT( * ) AS total
+			FROM {$wpdb->comments}
+			WHERE comment_type <> 'review' {$where}
```

Comment:
I think we might actually have to do a join on the posts table and do a `WHERE post_type <> 'product'` ?

(Because as you have it, I think replies to reviews would still be included?)